app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Here is a list of users</h1>
+<div class = 'user'>
+<h1> Welcome to Users List</h1>
+<div class = 'user_container'>
+<% @users.each do |user|%>
```

Comment:
- [optional] wonderful job. instead of using `@objects.each` i suggest organizing your work as such where you have the
```
 app/views/users/_user.html.erb
     ---- HTML Code Here displaying
     ---- user.photo, user.name and user.posts_counter
then
    app/views/users/index.html.erb
   ---- <%= render @users %>

```
this is a better practice.. nice job.